sql/python/README.md

```diff
@@ -1,13 +0,0 @@
-# Python Code Template
-
```

Comment:
I agree that we should merge the content of this file into build from source doc. but I am not sure the docker command in this file runs now. I'd suggest we confirm it works before moving it into the build from source file

Reply:
Will create an issue to track changes. For now, I will undo delete.